server/db/benchmarkQueries.sql

```diff
@@ -0,0 +1,58 @@
+-- psql -d reviewsdb -f ./db/benchmarkQueries.sql
+\timing
```

Comment:
When I was running my queries, the results would pop up in my terminal. I think the `\timing` flag was including the time that it took me to close the results in the terminal.

I switched to putting `explain analyze` in front of my queries which outputs info and execution time about the query.
https://www.educba.com/explain-analyze-in-postgresql/